config/environment.rb

```diff
@@ -1,6 +1,6 @@
 # Load the Rails application.
 require_relative 'application'
-
+require_relative './settings'
```

Comment:
This is causing problems. Go ahead and use ENV['FOO'] instead of settings for the omniauth stuff. Sorry to send you down a dead end street!

Reply:
That is because the settings are loaded in two different places. here and in [`settings.rb`](https://github.com/ContributorCovenant/beacon/blob/release/config/settings.rb)